processors/games/HungerGames.js

```diff
@@ -149,6 +150,7 @@ module.exports = ({
     wins: {
       solo: getKitStat(/^wins_(?!teams_)/),
       teams: getKitStat(/^wins_teams_/),
+      total: (getKitStat(/^wins_(?!teams_)/) + getKitStat(/^wins_teams_/)),
```

Comment:
This doesn't work - you should probably add the total in the code below (Line 184)